Parsers/ASimWebSession/Parsers/ASimWebSessionFortinetFortiGate.yaml

```diff
@@ -55,7 +55,7 @@ ParserQuery: |
     | extend 
         EventResultDetails = "NA"
```

Comment:
Can you help attaching sample logs to test the changes please:
![image](https://github.com/user-attachments/assets/b7ba220f-454d-4bf5-afe6-e0df2ca7bd38)

Reply:
https://learn.microsoft.com/en-us/AZURE/sentinel/normalization-develop-parsers#samples-submission-guidelines

Reply:
Apologies but i can not share sample events here. I will check the rest of the required changes.